ansible/group_vars/all.example

```diff
@@ -32,6 +32,16 @@ cartodb_user_admin_password: cartodb
 cartodb_user_email: cartodb@example.com
 cartodb_compile_assets: True
 
+# local account for common datasets; set cartodb_common_data_local to True to import to local account
+cartodb_common_data_local: False
+cartodb_common_data_user_email: commondata@example.com
+cartodb_common_data_user_password: cartodb
+cartodb_common_data_user_data_quota: 20.gigabytes
+
+# whether to import common datasets to local account as part of provision
+# (should only be true if cartodb_common_data_local is also true)
```

Comment:
Can we add a comment here to the effect of "Enabling this flag uses rake tasks not provided by the CartoDB repository. If enabled, you'll need to implement the rake tasks: <list of tasks>'.